packages/odyssey-react/src/components/Infobox/Infobox.theme.ts

```diff
@@ -14,20 +14,20 @@ import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
   // Root
-  FontLineLengthMax: theme.FontLineLengthMax,
-  SpacePadding: theme.SpaceRemM,
-  SpaceMarginBlockEnd: theme.SpaceRemM,
+  MaxFontLineLength: theme.FontLineLengthMax,
```

Comment:
You have a mismatch above with `FontLineLengthMax`. I'm calling this `MaxLineLength` in my open PRs.